Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,21 @@ class Program
     {
         static void Main(string[] args)
         {
-
+            var textLogger = new Logging.LogToConsole();
+            textLogger.LogOpenProgram();
+            try
+            {
+                int b = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'b'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5150047)